jupyterhub/user.py

```diff
@@ -238,6 +238,14 @@ def count_active_users(self):
         return counts
 
 
+def getRoleNames(role_list):
+    return [role['name'] for role in role_list]
+
+
+def getRoleObject(role_list, role_name):
+    return [role for role in role_list if role['name'] == role_name][0]
```

Comment:
Python standards don't use camelCase in Python names:

```suggestion
def _get_role_names(role_list):
    return [role['name'] for role in role_list]


def _get_role_object(role_list, role_name):
    return [role for role in role_list if role['name'] == role_name][0]
```